lua/dap/repl.lua

```diff
@@ -245,8 +249,7 @@ function execute(text)
     return
   elseif vim.tbl_contains(M.commands.clear, text) then
     if repl.buf and api.nvim_buf_is_loaded(repl.buf) then
-      local layer = ui.layer(repl.buf)
-      layer.render({}, tostring, {}, 0, - 1)
+      M.clear_output()
```

Comment:
```suggestion
      M.clear()
```

No reason the break the API